entry_tasks/16_Pick_movie_by _genre_or_actor/main.py

```diff
@@ -0,0 +1,76 @@
+if __name__ == "__main__":
+    # GENRES = []
+    a = input("Start searching by genre: ").strip()
+    genres_data = {}
+    genres_data['comedy'] = ['Meet the Parents', 'Anger Management']
+    genres_data['adventures'] = ['Mummy']
+    genres_data['romantic'] = ['Vanilla Sky', 'Meet Joe Black']
+    genres_data['drama'] = ['Meet Joe Black']
+    genres_data['thriller'] = ['Vanilla Sky']
+    genres_data['action'] = ['Mission Impossible']
+    genres_keys = list(genres_data.keys())
+    gk = f'{">>> Available Genres: "} {genres_keys}'
+    print(gk)
+    gki = genres_keys[0]
+    dict_values = genres_data.values()
+    b = input("Enter specific genre: ").strip()
+    if b.lower() == 'comedy':
+        dict_values = genres_data['comedy']
+        print("Available movies: ", dict_values)
+        c = input("Enter movie: ").strip()
+        if c.lower() == dict_values[0].lower():
```

Comment:
why not to check` if c in dict_values` instead of comparing all elements

Reply:
And you can add extra movies later! Assume this is your DB!

So what, you will be adding extra ifs like 
```
elif c == dict_values[2]
elif c == dict_values[3]
elif c == dict_values[4]
elif c == dict_values[infinite]
```
Indexes should not be hardcoded! You should not check by each index like that (in worst case, you need to loop over dict_values and compare elements, it will cover adding more movies later on at least)

Reply:
same problems below..

Reply:
@mykytapavlov Done.